elastic-agent-client.proto

```diff
@@ -185,9 +185,32 @@ message FQDNFeature {
     bool enabled = 1;
 }
 
+// Elastic APM TLS config
+message ElasticAPMTLS {
```

Comment:
Would it make sense to make this is more generic `TLS` message?

Reply:
Not sure if a generic TLS message will have the same format so I kept it only for Elastic APM in this case...